### ISSUE TITLE
W: non-break-space is good for what?

### ISSUE THREAD

**Issue by jengelh** [rpm-software-management/rpmlint#60]
```
'non-break-space',
'''The spec file contains a non-break space, which looks like a regular space
in some editors but can lead to obscure errors. It should be replaced by a
regular space.''',
```
rpmlint lacks a proper description of what these "obscure errors" are, and which program supposedly exhibits those.

**Comment by scop** [comment]
It's impossible to enumerate the possibilities, but here's just a few off the cuff:

In shell code:
```
$ ls -l
bash: ls -l: command not found
```

Write a non-breaking space between ```%setup``` and its arguments in a rpm specfile and watch the arguments being ignored.

Feel free to suggest better wording for the info message, preferably in the form of a pull request.

**Comment by jengelh** [comment]
rpmlint just should not warn about the nbsp when it occurs outside of shell, in particular, Summary: and %description.
